spec/tokenizer/errors/attribute_does_not_exist_spec.rb

```diff
@@ -0,0 +1,32 @@
+require './src/tokenizer/lexer.rb'
+require './src/tokenizer/errors.rb'
+
+require './spec/contexts/lexer.rb'
+require './spec/contexts/errors.rb'
+
+include Tokenizer
+include Errors
+
+RSpec.describe Lexer, 'error handling' do
+  include_context 'lexer'
+  include_context 'errors'
+
+  describe '#next_token' do
+    it 'raises an error when property is invalid (attribute)' do
+      mock_reader(
+        "あれは 配列\n" \
+        "ほげは あれの ふが\n"
+      )
+      expect_error AttributeDoesNotExist
```

Comment:
Technically this is a v1.1.0 error... But the attribute type is checked elsewhere and Errors::ExperimentalFeature is raised, so it's okay.